conn.go

```diff
@@ -233,7 +233,11 @@ func (cn *conn) handlePgpass(o values) {
 	if _, ok := o["password"]; ok {
 		return
 	}
-	filename := os.Getenv("PGPASSFILE")
+	// Get passfile from the options, if empty, get it from envvar
```

Comment:
i think the correct place to handle the env var is here: https://github.com/lib/pq/blob/381d253611d666974d43dfa634d29fe16ea9e293/conn.go#L2009

then the logic after `parseEnviron` is called will automatically take care of making the connection string parameter override the env var: https://github.com/lib/pq/blob/381d253611d666974d43dfa634d29fe16ea9e293/connector.go#L61

Reply:
I see you are right, it makes sense. I didn't see that I focused in that function. Pushed a new commit.